src/Application/Messenger/DonationUpserted.php

```diff
@@ -8,14 +8,11 @@
 
 class DonationUpserted extends AbstractStateChanged implements MessageGroupAwareInterface
 {
-    private function __construct(public string $uuid, public array $jsonSnapshot)
+    public function __construct(public string $uuid, public array $jsonSnapshot)
```

Comment:
I see this has made a shared test helper 2 lines shorter – not totally sure that's a benefit worthwhile for losing a belt and braces guarantee that the message always comes from a real `Donation`, seems like a move away from value objects by default? But I may be missing other benefits I'm yet to get to or not understanding a wider goal?

Reply:
This change is left over by accident from when I thought Donations couldn't fetch mandates. I will undo.

Reply:
Undone in 0f5768e8